client/src/entrypoints/admin/page-chooser-modal.js

```diff
@@ -94,7 +94,10 @@ const PAGE_CHOOSER_MODAL_ONLOAD_HANDLERS = {
       /* pagination links within search results should be AJAX-fetched
       and the result loaded into .page-results (and ajaxified) */
       // eslint-disable-next-line func-names
-      $('.page-results a.navigate-pages', modal.body).on('click', function () {
+      $(
+        '.page-results a.navigate-pages, .page-results [data-breadcrumb-item] a',
+        modal.body,
+      ).on('click', function () {
```

Comment:
```suggestion
      ).on('click', () => {
```
Might not need a name if it's an arrow function